extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -99,7 +99,7 @@ function waitForEvent(env, name) {
   });
 }
 
-describe.configure().ifNewChrome().run('Bind', function() {
+describe.configure().ifIe().ifNewChrome().run('Bind', function() {
```

Comment:
Reverted this file, as the change was unnecessary. Enabling this test on IE and on Chrome will require us to skip all other platforms. Happy to add that code here, or do it in another PR.